src/gsi/crtm_interface.f90

```diff
@@ -667,6 +667,21 @@ subroutine init_crtm(init_pass,mype_diaghdr,mype,nchanl,nreal,isis,obstype,radmo
    error_status = crtm_channelinfo_subset(channelinfo(1), &
         channel_subset = nuchan(subset_start:subset_end))
 
+!TODO JAJ the correct one    else if (channelinfo(1)%sensor_id(1:7) == 'iasi-ng' .AND. isis(1:7) == 'iasi-ng') then
```

Comment:
@wx20jjung , I saw your explanation to @DavidHuber-NOAA regarding these `TODO` statements.  Thank you for answering Dave's question.  Most developers won't see your comment in this PR.  Would you please add commented text to `crtm_interface.f90` explaning why these line have `TODO` around them?  You can remove `JAJ` when you update the comment.

Reply:
@RussTreadon-NOAA The current code in crtm_interface.f90 will fail with the correct crtm file.  Should I add an .or. with the correct sensor ID similar to read_iasing.f90?  This would circumvent a potential failure.  The TODO comments could be removed anytime after the correct file was inline.

Reply:
@wx20jjung .  We do not need to change any code.  I'm just asking that we make the comments clearer.   As an example see what I'm suggesting for `src/gsi/read_iasing.f90`

Reply:
Changed comments as requested.